### PR TITLE
vagrant default to local ursula-infra sensu server

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

File: envs/vagrant/defaults.yml [hunk 1]
```diff
@@ -65,13 +65,61 @@ client:
 monitoring:
   sensu_package_version: 0.12.5-1
   rabbit:
-    host: "{{ hostvars[groups['controller'][0]][primary_interface]['ipv4']['address'] }}"
-    port: 5672
-    user: openstack
-    pass: asdf
+    host: 172.16.0.15
```
Comment (j2sol) [inline envs/vagrant/defaults.yml]: This seems magic, is that okay?

File: envs/vagrant/defaults.yml [hunk 2]
```diff
@@ -65,13 +65,61 @@ client:
...
+    KBrBg9mzWtC6KQGwtt3RRInFycTYE6d+uf8OBKkCgYEAv8PNXbC4MEOfMWO8kJDd
+    xzxf+ana58SqXZfa+2H3j2Hco4wRioHDJzCQI6G4HO+QUV8jK+5jbW1N/Jgmke4I
+    g65XE6/Xh7GeUWghVFMNbAfpvRvbC/BLo/bipMZ4vas3otJa8gRo9sMJPPCUbl4J
+    9761jfvU1r80pH1JpvQ4hJUCgYAS+zpHe3dlxtRcilChoRo4gbRH0rIDK1oUoe79
+    NGovVwwqssGvFcQeTYD7odPwHnrWZJwDFfidNIq//M9wg/TdlvHetdSWs5qR5dGE
+    HpPepyo7f41aSi8CEt1smWjZcgYEapNq5VKno6Cd9V5a7V/HjtXLQfQfOG8gcM6l
+    TGHaKQKBgFRnRUE3tBeEKvM1touGxNjURea4aFVzFk4b8lL/OQJnkWhMmhrBKCHw
+    WRiRItwHQxTunlN0gK4qty2rczWIv/yzWALHJAOFLIyX0ynO4xY7x/3H9df9IuDe
+    Gs1548GcEmUG0CWujFXsEJkjjBxT4/fabx4NOJjg9Ij1Mi3LYmB2
+    -----END RSA PRIVATE KEY-----
```
Comment (j2sol) [inline envs/vagrant/defaults.yml]: Is this an unrelated change? copying the new cert I put in for example/ ?

Comment (paulczar) [inline envs/vagrant/defaults.yml]: nope, this is the cert set that I generated for the rabbit server in ursula-infra

**Comment by j2sol** [comment]
I remove my objections, :+1:
